### ISSUE TITLE
Plugins Colliding each other

### ISSUE THREAD

**Issue by Taninty** [ScruffyFurn/Windows-Unity-Ad-Network-Plugin#11]
![screen shot 2015-12-12 at 19 17 53](https://cloud.githubusercontent.com/assets/16267698/11762230/52df4cc6-a105-11e5-95f3-43f5529832fd.png)

I am taking build for Win phone 8.1 But I am getting this error. Could you help me to solve this out

**Comment by 2percenter** [comment]
Make sure you carefully read through which folders need which settings on the plugins. You need a copy of the plugin in your Assets\Plugins folder -- which you seem to have. Then you need a copy in an Assets\Plugins\Metro folder as well. Then make your settings appear as shown in this screenshot.

![wp_plugin](https://cloud.githubusercontent.com/assets/14305171/11971006/7ac5ea9e-a8fb-11e5-82c9-4c20b75308ce.PNG)

If that doesn't work, let us know. I struggled a lot with these, but the main thing I had to do was read and re-read the instructions... there are a lot of small details that are easy to miss.

**Comment by Taninty** [comment]
Thanks I solved this issue a days ago. I currently facing problem with debug mode of windows_adplugin. I have already seen in one issue to solve this. But I don't know how to implement release version for windows ad plugin. Could you help on that

**Comment by 2percenter** [comment]
I believe I had the same issue. What you need to get the release version of the plugin is already included. Within the files you downloaded, there is a Visual Studios Solution folder. Open the solution within in Visual Studio and create a released version. Then in your game's Visual Studio project, add that version of the Windows_Ad_Plugin.dll as a reference instead of what was brought over from Unity automatically.

I hope this at least gets you started in the right direction.

**Comment by Taninty** [comment]
Problem is from the example visual studio how will we create released version of dll. What i tried is i directly  linked their exsple dll and it shows its in debug mode

**Comment by 2percenter** [comment]
RIght. You have to open the Visual Studio project for the actual WIndows_Ad_Plugin. It is included in the files you probably downloaded. From Visual Studio, you can build the Windows_Ad_Plugin in Release mode. Then reference the new Release version of the plugin you created instead of the one you have been referencing.

**Comment by Taninty** [comment]
Thanks a lot. I try it and tell you.
